platform-sdk/swirlds-jasperdb/src/main/resources/META-INF/services/com.swirlds.virtualmap.datasource.VirtualDataSourceBuilder

```diff
@@ -0,0 +1 @@
+com.swirlds.merkledb.MerkleDbDataSourceBuilder
```

Comment:
Instead of handling the services file on our own we started in base to move to the Google Auto annotations. See:

https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-config-impl/build.gradle.kts
`mainModuleInfo { annotationProcessor("com.google.auto.service.processor") }`

https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConfigurationBuilderFactoryImpl.java
`@AutoService(ConfigurationBuilderFactory.class)`

We should try to have a common pattern in the repo. Open for discussion